deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -143,7 +143,7 @@ class ChatViewController: UITableViewController {
 
     private lazy var initialsBadge: InitialsBadge = {
         let badge: InitialsBadge
-        badge = InitialsBadge(size: 28, accessibilityLabel: String.localized("menu_view_profile"))
+        badge = InitialsBadge(size: 38, accessibilityLabel: String.localized("menu_view_profile"))
```

Comment:
the avatar appeared a bit too large to be pleasing, a tiny bit too few whitespace. i compared pixel-wise with telegram, and `37` seems what they're using, so we should go for that as well.

```suggestion
        badge = InitialsBadge(size: 37, accessibilityLabel: String.localized("menu_view_profile"))
```